util/processinfo.go

```diff
@@ -175,6 +175,8 @@ type SessionManager interface {
 	DeleteInternalSession(se interface{})
 	// Get all startTS of every transactions running in the current internal sessions
 	GetInternalSessionStartTSList() []uint64
+	// GetAutoAnalyzeID returns the auto analyze ID.
+	GetAutoAnalyzeID() uint64
```

Comment:
`GetAutoAnalyzeID` is not very appropriate for `SessionManager`. But I haven't come up with an elegant way to get `Server.globalConnID`.

Reply:
globalAutoAnalyzeID is made from serverID & reservedConnAnalyzeID, and serverID is Domain's serverID which is passed to Server with a wrapped serverIDGetter. What about passing serverIDGetter from Domain to Handle too and implementing a func like util.MakeID(serverIDGetter, util.reservedConnAnalyzeID)?
So that you don't need to put a new interface here.

Reply:
We can even think about moving globalConnID initialization from Server to Domain and passed to both Server and Handle, since both user connection and background jobs need it.

Reply:
Fixed. PTAL @chrysan @xhebox